Tests/configure-octopus-for-tentacle-tests.ps1

```diff
@@ -44,8 +44,19 @@ try
     $repository.Environments.Create($environment) | Out-Null
 
     #create a project
-    $projectGroup = $repository.ProjectGroups.FindByName("All projects")
+    #$projectGroup = $repository.ProjectGroups.FindByName("All projects")
```

Comment:
Can kill this

Reply:
I thought I'd already killed that line. Obviously not. Will fix and merge